source/standards/data_organization_in_our_lab.rst

```diff
@@ -0,0 +1,16 @@
+.. _ref_organization_standards:
```

Comment:
Additional information.

Start a project using raw data:
-> existing data check databases
-> data from open databases -> coordinate with arnaud
-> non existing data (meaning you acquire data)-> coordinate with arnaud

Start project using already preproc data
-> databases/your_database/derivatives
-> processedData/your_database/

Once you have run a pipeline coordinate with arnaud to put it in databases/your_database/derivative

Reply:
DO NOT duplicate data

Reply:
I think my way of presenting the different options is smoother and you are still missing some information here.